README.md

```diff
@@ -235,6 +235,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
 - [Shaarli](https://github.com/shaarli/Shaarli) - Personal, minimalist, super-fast, no-database bookmarking and link sharing platform. ([Demo](https://demo.shaarli.org)) `Zlib` `PHP`
 - [Shiori](https://github.com/RadhiFadlillah/shiori) - Simple bookmark manager built with Go. `MIT` `Go`
 - [unmark](https://github.com/plainmade/unmark) - Open source to do app for links. `MIT` `PHP`
+- [ubookmark](https://ungleich.ch/u/projects/ubookmark/) - LDAP enabled bookmarking service. ([Demo (IPv6 only)](https://ipv6-blog.place7.ungleich.ch/), [Source Code](https://ungleich.ch/u/projects/ubookmark/)) `GPL-2` `Python`
```

Comment:
Please remove the Demo (IPv6 only) link.
Please point the source code link to https://code.ungleich.ch/ungleich-public/ubookmark/
The correct license identifier is [GPL-2.0](https://spdx.org/licenses/GPL-2.0.html)

Reply:
Whats wrong with the demo link?